src/api/app/models/obs_factory/distribution.rb

```diff
@@ -109,7 +109,7 @@ def staging_projects_all
     def source_version
       Rails.cache.fetch("source_version_for_#{name}", expires_in: 10.minutes) do
         begin
-          p = Xmlhash.parse(ActiveXML::backend.direct_http "/source/#{name}/#{SOURCE_VERSION_FILE}?expand=1")
+          p = Xmlhash.parse(ActiveXML.backend.direct_http "/source/#{name}/#{SOURCE_VERSION_FILE}?expand=1")
```

Comment:
I know that this is already a big PR... but this ActiveXML calls must be move to Backen::Api calls